### PR TITLE
add httpd imagestream and template to the library

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
dependent on https://github.com/openshift/origin/pull/14660 before import can be run.

**Comment by bparees** [comment]
@coreydaley ptal

**Comment by bparees** [comment]
@sspeiche fyi

File: official.yaml [hunk 1]
```diff
@@ -68,6 +68,16 @@ data:
         docs: https://github.com/openshift/cakephp-ex/blob/master/README.md
       - location: https://raw.githubusercontent.com/openshift/cakephp-ex/master/openshift/templates/cakephp-mysql-persistent.json
         docs: https://github.com/openshift/cakephp-ex/blob/master/README.md
+  httpd:
+    imagestreams:
+      - location: https://raw.githubusercontent.com/openshift/origin/master/examples/image-streams/image-streams-rhel7.json
+        docs: https://github.com/sclorg/httpd-container/blob/master/README.md
+        regex: httpd
+        suffix: rhel7
+  httpd:
```
Comment (coreydaley) [inline official.yaml]: This httpd->templates should be combined with the above httpd just like nodejs (https://github.com/openshift/library/blob/master/official.yaml#L30-L42), there is no reason to have two separate top level httpd entries

**Comment by bparees** [comment]
@coreydaley fixed

**Comment by coreydaley** [comment]
@bparees lgtm

**Pull request merged**